Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -3986,7 +3985,9 @@ GstElement* MediaPlayerPrivateGStreamer::createVideoSink()
     if (!m_videoSink) {
         m_isUsingFallbackVideoSink = true;
         m_videoSink = webkitVideoSinkNew();
-        g_signal_connect_swapped(m_videoSink.get(), "repaint-requested", G_CALLBACK(repaintCallback), this);
+        g_signal_connect_swapped(m_videoSink.get(), "repaint-requested", G_CALLBACK(+[](MediaPlayerPrivateGStreamer* player, GstSample* sample) {
```

Comment:
You have the `+` character embedded there.

Reply:
Without it the build fails:

```
error: cannot cast from type '(lambda at /app/webkit/Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGSt
reamer.cpp:3988:91)' to pointer type 'GCallback' (aka 'void (*)()')                                                                                                                                                  
        g_signal_connect_data ((m_videoSink.get()), ("repaint-requested"), (((GCallback) ([](MediaPlayerPrivateGStreamer* player, GstSample* sample) { player->triggerRepaint(adoptGRef(sample)); }))), (this), __nul
l, G_CONNECT_SWAPPED);                                                                                                                                                                                               
                                                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                 
1 error generated.                                                                                 
```

Reply:
:man_shrugging:

Reply:
I would pass the lambda as std::function instead to avoid ambiguity.

Reply:
What is wrong with this anyway? We do  this in many places to convert lambdas to function pointers, expected by `G_CALLBACK()`. If this needs to change/improve, let's do it in another PR.

Reply:
```
rg "G_CALLBACK\(\+" Source/ | wc -l                                                                                                                                                           
105
```

Reply:
There's nothing wrong, and std::function wouldn't work anyway. I forgot about the + operator being needed when wrapping into G_CALLBACK.